src/Orleans/Serialization/RemoteNonDeserializableException.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Runtime.Serialization;
+using Orleans.Runtime;
+
+namespace Orleans.Serialization
+{
+    using System;
+    using System.Text;
+
+    /// <summary>
+    /// Represents an exception which cannot be fully deserialized.
+    /// </summary>
+    [Serializable]
+    public class RemoteNonDeserializableException : OrleansException
+    {
+        public RemoteNonDeserializableException() { }
+
+        /// <summary>
+        /// Gets the type name of the original <see cref="Exception"/> represented by this instance.
+        /// </summary>
+        public string OriginalTypeName { get; internal set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the original type of this exception can be reconstructed during
+        /// deserialization.
+        /// </summary>
+        internal bool CanAttemptReconstruction { get; set; }
```

Comment:
Ah, that field should have been removed. It's not used and isn't needed. Thanks for pointing this out.